Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM centos:centos7
+
+RUN yum install -y python3-pip && pip3 install htcondor
+
+COPY . /src/
+
+ENTRYPOINT ["/bin/bash"]
```

Comment:
We don't actually want this file in the main repo, you can leave it as an untracked file in your local copy